include/Request/Request.hpp

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <cstdlib>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "WebServerException/WebServerException.hpp"
+
+#define TRUE 1
+#define FALSE 0
```

Comment:
Why did you redefine these variables?

Reply:
They are used in the validateMethod function

Reply:
So... why don't use the native true and false?

Reply:
Because